rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/activity/CovidJsonFormActivity.java

```diff
@@ -64,6 +65,12 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
+    @Override
+    protected void onResume() {
+        super.onResume();
+        Utils.verifyUserAuthorization();
```

Comment:
Let's not do this for the jsonformactivity, this will be called too many times on every form launch (which I don't believe is necessary).

Reply:
done